### DISCUSSION TITLE
Where is the third stream ?

### DISCUSSION THREAD

**Discussion by The-NinthWave** [blakeblackshear/frigate#5345] [Ask A Question] [answered]
According to the specs: https://www.hikvision.com/ca-en/products/IP-Products/Network-Cameras/Pro-Series-EasyIP-/ds-2cd2143g2-iu/; this camera has 3 streams.

However, the management GUI shows only : Main and Sub

An ffprobe like: 
```
ffprobe -rtsp_transport tcp rtsp://nick:secret@10.0.40.10:554/Streaming/Channels/101
ffprobe -rtsp_transport tcp rtsp://nick:secret@10.0.40.10:554/Streaming/Channels/102
```
Returns an output while:

`ffprobe -rtsp_transport tcp rtsp://nick:tqwO4UuFa5Lc3Y0e@10.0.40.10:554/Streaming/Channels/103
`
Returns an error message.

Is my camera a ripoff or is just me who does not now how to access the third stream ?

**Comment by The-NinthWave** [comment, marked as answer]
Out of guess, I tried this

```
ffprobe -rtsp_transport tcp rtsp://nick:tqwO4UuFa5Lc3Y0e@10.0.40.10:554/Streaming/Channels/2

Input #0, rtsp, from 'rtsp://nick:tqwO4UuFa5Lc3Y0e@10.0.40.10:554/Streaming/Channels/2':
  Metadata:
    title           : Media Presentation
  Duration: N/A, start: 0.034000, bitrate: N/A
  Stream #0:0: Video: h264 (Main), yuv420p(progressive), 352x240, 30 fps, 30 tbr, 90k tbn
```

Does it solve the problem?
Can I make use of this stream ?

**Reply by sunnysingh-84** [reply]
3rd stream on hikvision is only available in certain conditions, from memory vca resource needs to be changed from smart events to monitor and I think you cant use h265 codecs
